src/Haskell/Ide/Engine/Scheduler.hs

```diff
@@ -296,21 +297,23 @@ ghcDispatcher
   -> Channel.OutChan (GhcRequest m)
   -> IdeGhcM void
 ghcDispatcher env@DispatcherEnv { docVersionTVar } errorHandler callbackHandler pin
-  = forever $ do
+  = do
+  iniDynFlags <- getSessionDynFlags
```

Comment:
I think this can be removed. It's an old hack.

Reply:
Can we just get the session dyn flags in `loadCradle` as we need it?